### PR TITLE
Allowing only UTC-compatible VM timezones

### LINKED ISSUE #93: Impossible to start the final VM when clock.timezone is specified
hint from vm-import-operator log `No matching template was found for the virtual machine using empty vm definition"`

kubevirt error:

```js
server error. command SyncVMI failed: "LibvirtError(Code=1, Domain=20, Message='internal error: missing 'timezone' attribute for clock with offset='timezone'')"
```

vm

```YAML
apiVersion: kubevirt.io/v1alpha3
kind: VirtualMachine
metadata:
  annotations:
    comment: ''
    kubevirt.io/latest-observed-api-version: v1alpha3
    kubevirt.io/storage-observed-api-version: v1alpha3
    sso: disabled
  selfLink: /apis/kubevirt.io/v1alpha3/namespaces/default/virtualmachines/acirrosvm
  resourceVersion: '868303'
  name: acirrosvm
  uid: 21cba2e8-b8f4-4d27-a807-4cb1521c5247
  creationTimestamp: '2020-04-15T17:19:59Z'
  generation: 4
  namespace: default
  finalizers:
    - k8s.v1.cni.cncf.io/kubeMacPool
  labels:
    origin: ovirt
    tags: ''
spec:
  running: true
  template:
    metadata:
      creationTimestamp: null
    spec:
      domain:
        clock:
          timer: null
          timezone: Etc/GMT
        cpu:
          cores: 1
          sockets: 1
          threads: 1
        devices:
          autoattachGraphicsDevice: true
          disks:
            - disk:
                bus: virtio
              name: disk0
        firmware:
          bootloader:
            bios: {}
          serial: c1a784d6-4747-4c26-8b47-d3d6766436e1
          uuid: 5b6203fc-5acd-56a3-a3e1-f07bbf2cb8d5
        machine:
          type: q35
        memory: {}
        resources:
          limits:
            memory: '4294967296'
          requests:
            memory: '1072693248'
      volumes:
        - name: disk0
          persistentVolumeClaim:
            claimName: 64302e7f-3f08-4d32-9fe1-59b6c383acb5
status:
  created: true
```

### PR DESCRIPTION
**Pull request by jakub-dzon**
Fixes #93
```release-note
NONE
```
Signed-off-by: Jakub Dzon <jdzon@redhat.com>

### CODE REVIEW

File: pkg/providers/ovirt/mapper/mapper.go [hunk 1]
```diff
@@ -594,11 +594,10 @@ func (o *OvirtMapper) mapPlacementPolicy() *kubevirtv1.EvictionStrategy {
...
 
 func (o *OvirtMapper) mapTimeZone() *kubevirtv1.Clock {
-	clock := &kubevirtv1.Clock{}
-	if timezone, ok := o.vm.TimeZone(); ok {
-		timezoneName, _ := timezone.Name()
-		clockOffsetTimezone := kubevirtv1.ClockOffsetTimezone(timezoneName)
-		clock.Timezone = &clockOffsetTimezone
+	clock := kubevirtv1.Clock{
+		ClockOffset: kubevirtv1.ClockOffset{
+			UTC: &kubevirtv1.ClockOffsetUTC{},
```
Comment (pkliczewski) [inline pkg/providers/ovirt/mapper/mapper.go:599]: Why not to use `o.vm.UtcOffset()` here?

Comment (jakub-dzon) [reply pkg/providers/ovirt/mapper/mapper.go:599]: Good point. I think it can be used if present.

Comment (jakub-dzon) [reply pkg/providers/ovirt/mapper/mapper.go:599]: Actually - I don't see a way to make this field non-null from oVirt UI (where our cirrosvm lives). Do you know how to do it (besides API)?

Comment (pkliczewski) [reply pkg/providers/ovirt/mapper/mapper.go:599]: No, I only checked the API whether there is something we can use.

**Review by pkliczewski — APPROVED** [review]

**Pull request merged**